### ISSUE TITLE
router.subscribe() called multiple times

### ISSUE THREAD

**Issue by frederikhors** [pateketrueke/yrv#27]
Using this code with multiple nested `<Router>`: https://svelte.dev/repl/8ae743aa80214ed696da8d13df392aa7?version=3.20.1

I have `router.subscribe()` called multiple times:

![image](https://user-images.githubusercontent.com/41120635/78778396-8070f680-799b-11ea-90bd-ab6f2fb1f649.png)

**Comment by pateketrueke** [comment]
Interesting, related to #16 right?

**Comment by frederikhors** [comment]
> Interesting, related to #16 right?

Yep.

**Comment by pateketrueke** [comment]
I found that any subscription to stores will fire initially as they have an initial value, then once the router found something updates its info and a second event is fired.

So is completely normal to get two events on the initial load, what I did was adding an `initial` property on the router-info to help you identify this initial event.

Also, I tried to make fewer changes on the `$router` store to keep events at minimum.
